README.md

```diff
@@ -1,64 +1,8 @@
 401 JS --  Lab 03 file system
```

Comment:
Just making a top comment to let you know your package.json is not in this directory ```lab-devon```. This may cause problems in the future, so make sure your package.json is directly in your ```lab-devon``` directory, or whatever you make your root lab directory.